tests/installation/accept_license.pm

```diff
@@ -35,16 +35,7 @@ sub run {
     push @tags, 'inst-welcome-no-product-list' if is_sle('15+') && get_var('UPGRADE');
 
     assert_screen \@tags;
-    if (match_has_tag('network-settings-button')) {
-        # workaround for hpc missing license: https://bugzilla.suse.com/show_bug.cgi?id=1060174
-        if (check_var('SLE_PRODUCT', 'hpc')) {
-            record_soft_failure('bsc#1060174');
-            return;
-        }
-        else {
-            die 'It seems that license agreement is missing, please check!';
-        }
-    }
+    die 'It seems that license agreement is missing, please check!' if match_has_tag('network-settings-button');
```

Comment:
the tag is still in use here

Reply:
Ah, my bad, missed this point. Could you please then undo line #64 removal and I'll merge. Thanks!

Reply:
done